src/main/java/com/wiley/autotest/selenium/elements/upgrade/VisibleTeasyElement.java

```diff
@@ -0,0 +1,41 @@
+package com.wiley.autotest.selenium.elements.upgrade;
+
+import com.wiley.autotest.selenium.elements.upgrade.v3.ElementWaitFor;
+import com.wiley.autotest.selenium.elements.upgrade.v3.Should;
+import com.wiley.autotest.selenium.elements.upgrade.v3.VisibleElementWaitFor;
+import com.wiley.autotest.selenium.elements.upgrade.v3.VisibleShould;
+import com.wiley.autotest.selenium.context.SearchStrategy;
+
+import static com.wiley.autotest.utils.TestUtils.waitForSomeTime;
+
+
+/**
+ * Represents element that is visible for the user
```

Comment:
Compare with the description upper
'+ * VISIBLE - element that is displayed to the user'

Reply:
corrected